internal/checker/cloudformation/rules/networking/verify_security_group_port_22.rego

```diff
@@ -0,0 +1,52 @@
+# METADATA
+# title: "Verify Security Groups allowing ingress from 0.0.0.0:0 to port 22"
+# description: "Checks if the security groups are allowing ingress from 0.0.0.0/0 to port 22"
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-securitygroup.html
+# custom:
+#   id: CB_CFT_014
+#   severity: LOW
+package lib.cloudformation.CB_CFT_014
+
+import future.keywords.in
+
+resource := ["AWS::EC2::SecurityGroup", "AWS::EC2::SecurityGroupIngress"]
+
+is_valid {
+	some resources in input.Resources
+	some resource_type in resource
+	resources.Type == resource_type
+}
+
```

Comment:
swap logic for pass and fail so that it will only fail when ingress allow "0.0.0.0/0"

fail[ingress] {
	is_valid
	some resources in input.Resources
	some ingress in resources.Properties.SecurityGroupIngress
	ingress.FromPort == 22
	ingress.ToPort == 22
	ingress.CidrIp == "0.0.0.0/24"
}

pass[resources] {
	some resources in input.Resources
	is_valid
	count(fail) == 0
}